pkg/data/egress_lists/egress_lists.go

```diff
@@ -100,12 +111,42 @@ func EgressListToString(egressListYamlStr string, variables map[string]string) (
 				tlsDisabledURLListStr += urlStr
 				continue
 			}
+
+			// Egress URL from OCM or CLI passed as slice, and converted to a joint string by comma
+			if strings.Contains(urlStr, ",") {
```

Comment:
Why is this necessary here?Shouldn't it be one URL per input that is looped on line 97?

Reply:
in all the existing templates, it converts the URL based on the given paraemeters, like AWS_REGION, and do a one to one converting, means each item in the template will be converted to one testing URL

But for the ECR registry, one requirement is the ECR registry can be multiple. And it may not be a fixed length. So we have to do with the one item from the template to multiple URLs converting.

The easiest way I found is just convert the passed ECR slice to string and connect with `,` and then call the new function `ConvertECRRegistryURLs` to conver the joint string to separated URLs